tests/test_sns.py

```diff
@@ -611,3 +611,92 @@ def test_sns_delete(self):
         client = session_factory().client('sns')
         resources = client.list_topics()['Topics']
         self.assertEqual(len(resources), 0)
+
+    def test_sns_tag(self):
+        session_factory = self.replay_flight_data("test_sns_tag")
+        p = self.load_policy(
+            {
+                "name": "tag-sns",
+                "resource": "sns",
+                "filters": [{"tag:Tagging": "absent"}],
+                "actions": [{"type": "tag", "key": "Tagging", "value": "added"}],
+            },
+            session_factory=session_factory,
+        )
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+
+        client = session_factory().client("sns")
+        tags = client.list_tags_for_resource(ResourceArn=resources[0]["TopicArn"])["Tags"]
+        self.assertEqual(tags[0]["Value"], "added")
+
+    def test_sns_marked_for_op_tag(self):
```

Comment:
its not really needed to write a test exercising the marked-for-op functionality in isolation since its common to all resources, and has dozen of other tests, it suffices to simply verify the action registry registration here.

Reply:
Understood! Combined it with the remove tag test! Thanks for the feedback